upi/metal/config.tf

```
@@ -176,6 +176,36 @@ EOF
 
 }
 
+variable "packet_plan" {
+  type    = string
+  default = "c1.small.x86"
+
+  description = <<EOF
+The Packet.Net device plan slug.
+EOF
+
+}
+
+variable "packet_facility" {
+  type    = string
+  default = "any"
+
+  description = <<EOF
+The Packet.Net facilities code(s) to be used.
```

nit: this can only be one code
```suggestion
The Packet.Net facilities code to be used.
```

---

Fixed.